demos/01 Unsupervised Classification/01a Training an Unsupervised Classifier.ipynb

```diff
@@ -303,20 +289,20 @@
     "try:\n",
     "    # Create a new Blob object\n",
     "    blob = Blob(\n",
-    "        name=\"training_kmeans_model\",\n",
+    "        name=f\"training_kmeans_model\",\n",
     "        tags=[\"ml-examples\"],\n",
     "    )\n",
     "    # Upload our model to this Blob:\n",
     "    blob.upload(\"training_kmeans.pickle\")\n",
     "except:\n",
     "    print(\"Blob already exists, deleting old iteration:\")\n",
     "    # Already exists, overwriting\n",
-    "    blob = Blob.get(name=\"training_kmeans_model\", namespace=f\"{org}:{user_id}\")\n",
```

Comment:
Is the namespace arg here removed because it's superfluous? Just want to make sure we're not stepping on other people running this notebook

Reply:
good call. I will re-add it with the improved-legibility namespace formatted string that we talked about on slack